packages/client/lib/sync/txpool.ts

```diff
@@ -325,7 +359,7 @@ export class TxPool {
 
     const newTxHashes = []
     for (const tx of txs) {
-      this.add(tx)
+      await this.add(tx)
```

Comment:
here (and below on L418) you can add try catch to `await this.add(tx)`, and in the catch just do a ```logger.info(`Error adding tx to TxPool: ${error.message}`)``` and continue, not much else we need to do if we get a bad tx from devp2p. we could do a `logger.error`, but i think that is too verbose, for something that is out of our control.